### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by cvrle77** [muraliwebworld/push-notification-plugin-for-wordpress-websites#9] [Ideas]
Can we please please please get Scheduler directly as an option in every single post (Edit post)? This way, when we write, or edit post, we can set the date, hit schedule, and that is all. This way we can add quickly and easily 5-10-50 posts if necessary, without the need to search link, or image, or to type excerpt, when plugin already does that beautifully. There would be no need for creators to leave their writing environment to schedule post, or have any technical skills to do this task. 

Then, scheduled posts in this manner, should appear in Notifications tab, where we can check everything, or edit, if necessary.
I made some edits to show you visually what I had in mind.

![image](https://github.com/muraliwebworld/push-notification-plugin-for-wordpress-websites/assets/16167396/a2a3b490-b39b-4f89-b987-8ca64574daf5)

**Comment by muraliwebworld** [comment]
I will add this feature in one of future release soon, i will update you. I have also checked your issue on scheduling, if schedule button is ON then it will schedule post other if you want to cancel the schedule then switch off the schedule button, it will automatically cancel the schedule in action scheduler. Action scheduler plugin is merged with PNFPB plugin so PNFPB uses action scheduler (not normal cron jobs ...action scheduler also look like cron but it handles more load using queue ....one of my client said they were able to process more than 90000 tokens using this plugin...due to this action scheduler is used in PNFPB plugin). Even if you cancel schedule in action scheduler it will be cancelled , i have also checked today. I will be releasing newer version this week with other changes, i need to update post notifications because gutenberg editor is triggering save post action 2 times, due to this notification occurs 2 times, it will be rectified in this week release but i am not sure whether i can include schedule option inside the post, i will try to include this option in next week release. Also I am working on PWA enhancements in this plugin for this week's release. If possible i will try to include scheduling inside post editor in this week release.
